packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -654,6 +654,12 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
   SemanticsHandle ensureSemantics() {
     return binding.pipelineOwner.ensureSemantics();
   }
+
+  /// Given a [Scrollable] widget specified by [finder], scrolls the widget
```

Comment:
The distinction between the widget being made visible and the scrollable is a little confusing in the wording here.  How about:

```dart
/// Given a widget `W` specified by [finder] and a [Scrollable] widget `S` in
/// its ancestry tree, this scrolls `S` so as to make `W` visible.
///
/// Shorthand for `Scrollable.ensureVisible(tester.element(finder))`
```

Reply:
Thanks, Yours is simpler. I'll update it.